internal/transport/transport.go

```diff
@@ -302,6 +304,10 @@ func (cfg *Config) validate() error {
 		return ErrInvalidRateLimit
 	}
 
+	if cfg.ConnectionStrings == nil {
+		cfg.Logger.Warn("No connectionStrings specified in the config file.")
```

Comment:
Can you wrap the message, like [this](https://github.com/alpine-hodler/gidari/blob/main/internal/transport/transport.go#L275)?

Reply:
Sure. 1 second :)

Reply:
Done.

Reply:
@rangzen There was a minor linting failure. Don't worry about the build failure, it's unrelated and I will make a separate issue for that.

Reply:
Linting failure fixed. Sorry...